etc/openstackquickstartrc

```diff
@@ -29,6 +29,9 @@ with_horizon=yes
 # Setup Manila
 with_manila=yes
 
+# Setup Magnum
+with_magnum=yes
```

Comment:
I would prefer to have it enabled. Then we at least test the package installation (even if we currently don't setup and run the tempest tests)